tox.ini

```diff
@@ -36,12 +37,14 @@ basepython = python3.11
 
 [testenv:py3-unit]
 basepython = {[testenv:py3]basepython}
+setenv = {[testenv]setenv}
```

Comment:
isn't the default `testenv` section value already inherited here? why do we need to add these two lines?

Reply:
Ah, that's an artifact of my debugging. I agree. Let's remove it.

Reply:
Removed